static/templates/confirm_mute_user.hbs

```diff
@@ -0,0 +1,5 @@
+<p>
+    {{#tr this}}
+    Would you like to mute user <b>__user_name__</b>?  This will mark all received messages from __user_name__ as read, and subsequent messages sent by __user_name__ while muted will be marked as read automatically.
```

Comment:
We may want to avoid mentioning the user's name 3 times; e.g.:
```
Would you like to mute user <b>__user_name__</b>?  Messages sent by muted users will never trigger notifications, will be marked as read, and will be hidden."
```

(We should also use a `help_link_widget` somewhere on this dialog to link to the /help/ article).

Reply:
Makes sends, done.